README.md

```diff
@@ -10,12 +10,16 @@ Under construction!
 1. Make a virtual environment and source into it
 1. `pip install -r requirements.dev.txt`
 1. `npm install -g npm-watch`
+1. `npm install -g riot`
+1. `npm install -g stylus`
 1. `brew install elasticsearch` (defaults should work)
 
 
 ## Running it locally
 
 
+`elasticsearch` to start running elasticsearch before running the server.
```

Comment:
I think we should make this mac specific, how do you run ES in the background? I think `brew install elasticsearch` has instructions for this?

Reply:
Not sure what you are asking? running the command `elasticsearch` should spool up an instance (if that's what it would be considered) of whatever voodoo Elasticsearch does. I believe the final line of the `brew` command says something about running Elasticsearch, yes, so do we drop this line entirely as duplicate instructions or ?

Reply:
I think we can add some instructions about "Hey, warning, you need to have ES running for this. To run ES with mac, install it with `brew install elasticsearch` and make sure you start the ES server after installing it"